examples/plugins/org.jboss.tools.project.examples/src/org/jboss/tools/project/examples/model/AbstractImportProjectExample.java

```diff
@@ -57,11 +57,11 @@ public IPath getLocation() {
 			String location = ProjectExamplesActivator.getDefault().getPreferenceStore().
 					getString(ProjectExamplesActivator.PROJECT_EXAMPLES_OUTPUT_DIRECTORY);
 			if (location == null || location.isEmpty()) {
-				return Platform.getLocation();
+				return Platform.getLocation().append("examples");

```

Comment:
i know its a longshot but what happens in case of someone having a project called "examples" in his workspace ?